utils/tests_fetcher.py

```diff
@@ -704,33 +669,54 @@ def filter_tests(output_file, filters):
         help="Will only print the tree of modules depending on the file passed.",
         default=None,
     )
+    parser.add_argument(
+        "--commit_message",
+        type=str,
+        help="The commit message (which could contain a command to force all tests or skip the CI).",
+        default=None,
+    )
     args = parser.parse_args()
     if args.print_dependencies_of is not None:
         print_tree_deps_of(args.print_dependencies_of)
-    elif args.sanity_check:
-        sanity_check()
     elif args.filter_tests:
         filter_tests(args.output_file, ["pipelines", "repo_utils"])
     else:
-        repo = Repo(PATH_TO_TRANFORMERS)
+        repo = Repo(PATH_TO_REPO)
+        commit_message = repo.head.commit.message
+        commit_flags = parse_commit_message(commit_message)
+        if commit_flags["skip"]:
+            print("Force-skipping the CI")
+            quit()
+        if commit_flags["test_all_models"]:
+            print("Testing all models found.")
+        if commit_flags["test_all"]:
+            print("Force- launching all tests")
```

Comment:
When reading this part, I feel confused between between `"test_all_models"` and `"test_all"`.
Maybe `run_fetched_tests` and `run_all_tests` make it more clear. 
Leave you to decide.

(and `"Run all fetched tests.`)

Reply:
Going for no_filter instead, as I find it clearer and shorter than `run_fetched_tests`.